mws/acceptance/private_access_settings_test.go

```diff
@@ -0,0 +1,29 @@
+package acceptance
+
+import (
+	"os"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/internal/acceptance"
+)
+
+func TestMwsAccPrivateAccessSettings(t *testing.T) {
+	cloudEnv := os.Getenv("CLOUD_ENV")
+	if cloudEnv != "MWS" {
+		t.Skip("Cannot run test on non-MWS environment")
+	}
+	acceptance.Test(t, []acceptance.Step{
+		{
+			Template: `provider "databricks" {
```

Comment:
please remove 

```
provider "databricks" {
				host     = "{env.DATABRICKS_HOST}"
				username = "{env.DATABRICKS_USERNAME}"
				password = "{env.DATABRICKS_PASSWORD}"
			}
```

and see if test still passes

Reply:
Done